source/operations/cert-manager.rst

```diff
@@ -0,0 +1,577 @@
+.. _minio-certmanager:
+
+============
+cert-manager
+============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+TLS certificate management with cert-manager
+--------------------------------------------
+
+`cert-manager <https://cert-manager.io/>`__ works as a controller for managing certificates within Kubernetes clusters.
+cert-manager obtains valid certificates from an ``Issuer`` or ``ClusterIssuer`` and can renew certificates prior to expiration.
+
+The MinIO Operator supports using cert-manager for certificates as an alternative to the MinIO Operator managing certificates for itself and its tenants.
+
+On a Kubernetes cluster, cert-manager requires a global level ``ClusterIssuer`` to generate intermediary ``Issuers`` and certificates.
+At the namespace level, cert-manager issues certificates derived from an ``Issuer``.
+A ``ClusterIssuer`` can issue certificates for multiple namespaces.
+An ``Issuer`` only issues certificates for its own namespace.
+
+To learn more about cert-manager Issuers, refer to the `Issuer documentation <https://cert-manager.io/docs/concepts/issuer/>`__.
+
+Below is a logical view of a Kubernetes cluster with four namespaces:
+
+- ``minio-operator``
+- ``tenant-1``
+- ``tenant-2``
+- ``other-namespace``
+
+.. image:: /images/k8s/cert-manager-namespaces.svg
+   :width: 600px
+   :alt: A graphic depiction of a kubernetes cluster with four namespaces. Each namespace in its own white box. One is labeled minio-operator with contents two sts content items. A second is labeled tenant-1 with a MinIO logo and four drives of a pool. A third is labeled tenant-2 with similar contents to the tenant-1 box. The fourth is labeled other namespace with contents of other pods.
+   :align: center
+
+This guide shows you how to set up a different Certificate Authority (CA) in each namespace.
```

Comment:
I would jump straight to this.

Reply:
Moved to the top of the page.

Reply:
But keeping a chunk of the rest of the content for context. Because you need to install cert-manager, then install a certificate for cert-manager, the operator, and each tenant. Trying to explain all of that to set explectations.